modules/Sources/Dependencies/AddressBookClient/AddressBookLiveKey.swift

```diff
@@ -206,11 +214,11 @@ extension AddressBookClient: DependencyKey {
         return syncedContacts
     }
     
-    private static func storeContacts(
+    @discardableResult private static func storeContacts(
         _ abContacts: AddressBookContacts,
         remoteStorage: RemoteStorageClient,
         remoteStore: Bool = true
```

Comment:
It's clearer for this not to have a default.
```suggestion
        remoteStore: Bool
```